deltacat/compute/compactor_v2/model/hash_bucket_input.py

```diff
@@ -15,6 +15,7 @@ def of(
         primary_keys: List[str],
         num_hash_buckets: int,
         num_hash_groups: int,
+        hb_task_index: Optional[int] = 0,
```

Comment:
Is this passed only for logging? Isn't taskID sufficient for identifying logs?

Reply:
No, the problem is the memory allocation logs are emitted by the driver, whereas the actual hash bucketing logic is done by the worker. The workers currently do not emit which hash bucket index they are, so it is impossible (or at least much more difficult than it should be, if a way exists) to tell which worker corresponds to which memory allocation log.